apps/api/v2/src/ee/bookings/2024-08-13/services/bookings.service.ts

```diff
@@ -307,9 +307,26 @@ export class BookingsService_2024_08_13 {
 
     const bookingRequest = await this.inputService.createCancelBookingRequest(request, bookingUid, body);
     await handleCancelBooking(bookingRequest);
+
+    if ("cancelSubsequentBookings" in body && body.cancelSubsequentBookings) {
+      return this.getAllRecurringBookingsByIndividualUid(bookingUid);
+    }
+
     return this.getBooking(bookingUid);
   }
 
+  private async getAllRecurringBookingsByIndividualUid(bookingUid: string) {
```

Comment:
not sure about the naming of this function, reading it I would expect the function to return an array of bookings

Reply:
Maybe I should rename `this.getBooking` then, because `getAllRecurringBookingsByIndividualUid` will return an array of all recurrences withing a recurring bookings.

It calls  `this.getBooking` which then determines if its uid of individual booking or of the whole sequence.